cmake/external/ascend.cmake

```diff
@@ -65,6 +65,7 @@ endif()
 if(WITH_ASCEND_CL)
   set(ASCEND_CL_DIR ${ASCEND_DIR}/ascend-toolkit/latest/fwkacllib/lib64)
 
+  set(inf_eccl_lib /home/liuwei88/01_eccl/ECCL/build/lib/libeccl.so)
```

Comment:
let's try not use hard coded path in code

Reply:
how about use the same path as hccl.so, then we need to place this package there at advance?